mobileweb/titanium/Ti/UI/Animation.js

```diff
@@ -423,6 +423,7 @@ define(['Ti/_/declare', 'Ti/_/Evented', 'Ti/_/style', 'Ti/UI'], function(declare
 
 		promise.pause = function() {
 			var a = findAnimation();
+			updateRect(a.elem);
```

Comment:
We notice updateRect() is called multiple times. Is there a reason it can't be called once?